README.md

```diff
@@ -23,179 +25,55 @@ is an optional component that currently supports:
-
-Currently, only the following Linux distributions and versions are supported:
-
-- Amazon Linux: 2
-- CentOS / Red Hat / Oracle: 7, 8
-- Debian: 8, 9, 10
-- Ubuntu: 16.04, 18.04, 20.04
-
-### Standalone Windows MSI (64-bit only)
-
-A Windows MSI package is available to download at
-[https://github.com/signalfx/splunk-otel-collector/releases
-](https://github.com/signalfx/splunk-otel-collector/releases) for versions
-v0.4.0 or later.
-
-To install, double-click on the downloaded package or run the following command
-in a PowerShell terminal:
-
-```sh
-PS> Start-Process -Wait msiexec "/i PATH_TO_MSI /qn"
-```
-
-Replace `PATH_TO_MSI` with the *full* path to the downloaded package, e.g.
-`C:\your\download\folder\splunk-otel-collector-0.4.0-amd64.msi`.
-
-The collector will be installed to
-`\Program Files\Splunk\OpenTelemetry Collector`, and the
-`splunk-otel-collector` service will be created but not started.
-
-A default config file will be copied to
-`\ProgramData\Splunk\OpenTelemetry Collector\config.yaml` if it does not
-already exist.  This file is required to start the `splunk-otel-collector`
-service.
-
-Before starting the `splunk-otel-collector` service, the following variables
-in the default config file need to be replaced by the appropriate values for
-your environment:
-
-- `${SPLUNK_ACCESS_TOKEN}`
-- `${SPLUNK_REALM}`
-- `${SPLUNK_BALLAST_SIZE_MIB}`
-- `${SPLUNK_MEMORY_LIMIT_MIB}`
-- `${SPLUNK_MEMORY_SPIKE_MIB}`
-
-See the [Getting Started](#getting-started) section for details about these
-variables.
-
-After updating all variables in the config file, start the
-`splunk-otel-collector` service by rebooting the system or running the
-following command in a PowerShell terminal:
-
-```sh
-PS> Start-Service splunk-otel-collector
-```
-
-The collector logs and errors can be viewed in the Windows Event Viewer.
+The Collector is supported on and packaged for a variety of platforms including:
 
-### More examples
+- Kubernetes
+  - Helm (coming soon!)
+  - [YAML](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/exporter/sapmexporter/examples/signalfx-k8s.yaml)
+- Linux
+  - [Installer script (recommended)](./docs/getting-started/linux-installer.md)
+  - [Standalone](./docs/getting-started/linux-standalone.md)
+- Windows
+  - Install script (coming soon!)
+  - [Standalone](./docs/getting-started/windows-standalone.md)
 
 You can consult additional use cases in the [examples](./examples) directory.
 
+## Supported Components
+
+The distribution offers support for the following components. Support is based
+on the [OpenTelemetry maturity
+matrix](https://github.com/open-telemetry/community/blob/47813530864b9fe5a5146f466a58bd2bb94edc72/maturity-matrix.yaml#L57).
+
+### Beta
+
+| Receivers        | Processors        | Exporters | Extensions    |
+| :--------------: | :--------:        | :-------: | :--------:    |
+| fluentforward    | attributes        | file      | fluentbit     |
```

Comment:
We're not using fluentbit right now and hopefully will be replaced with Stanza. Any reason to support it?

Reply:
I suspect we do given log observer customers have opted for fluentbit -- perhaps we can have them go direct, but through collector can help with metadata. Do you see any risks here?

Reply:
let me move to alpha

Reply:
Are they actually using the extension though which runs the executable (which we don't ship with) or are they running fluentbit themselves and sending to us using fluentforward receiver?

Reply:
I am good dropping, but it is packaged currently -- can you file an issue and I will handle that in a separate PR?

Reply:
https://github.com/signalfx/splunk-otel-collector/issues/100